examples/google-cloud.md

```diff
@@ -0,0 +1,68 @@
+# Executando MariTalk Local na Google Cloud Platform (GCP)
+
+Este tutorial mostra como executar a MariTalk Local na Google Cloud Platform (GCP). Para isso, utilizaremos Docker em uma instância com a GPU NVIDIA L4 24GB.
+
+1. Crie uma instância na GCP selecionando NVIDIA L4 como __GPU type__ em __Machine configuration__.
```

Comment:
should we mention that the user should select ubuntu 20.02 image, so the user has a reference of which image to choose from?